kde-frameworks/kdelibs4support/kdelibs4support-5.109.0.ebuild

```diff
@@ -86,17 +87,13 @@ PATCHES=(
 
 src_prepare() {
 	ecm_src_prepare
-
-	if ! use handbook; then
-		sed -e "/kdoctools_install/ s/^/#DONT/" -i CMakeLists.txt || die
-	fi
-
 	cmake_run_in src cmake_comment_add_subdirectory l10n
 }
 
 src_configure() {
 	local mycmakeargs=(
 		-DWITH_X11=$(usex X)
+		-DCMAKE_DISABLE_FIND_PACKAGE_NetworkManager=ON
```

Comment:
Between https://github.com/gentoo/gentoo/pull/32474#discussion_r1306722535 and this change, did you come to a conclusion that this feature does not do anything (for us?) after all?

Reply:
> There is a standalone kded service generated if NM support is enabled
`kded_networkstatus` is always being generated, just with or without NM backend support.